src/main/drivers/sdcard_sdio_baremetal.c

```diff
@@ -399,6 +427,7 @@ bool sdcard_poll(void)
                 if (sdcard.multiWriteBlocksRemain > 1) {
                     sdcard.multiWriteBlocksRemain--;
                     sdcard.multiWriteNextBlock++;
+                    if (sdcard.useCache) cache_reset();
```

Comment:
Please use blocks even for single line conditional expressions. This makes code easier to maintain, and reduces the potential for bugs.